src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -593,14 +577,13 @@ private static string GetAadAuthorityAudienceValue(AadAuthorityAudience authorit
         /// Adds a known Authority corresponding to an ADFS server. See https://aka.ms/msal-net-adfs
         /// </summary>
         /// <param name="authorityUri">Authority URL for an ADFS server</param>
-        /// <param name="isDefaultAuthority">Boolean telling if this is the default authority
-        /// for the application</param>
+        /// <param name="validateAuthority">TODO(migration): documentation</param>
         /// <remarks>You can add several authorities, but only one can be the default authority.
         /// MSAL.NET will only support ADFS 2019 or later.</remarks>
         /// <returns>The builder to chain the .With methods</returns>
-        public T AddKnownAdfsAuthority(string authorityUri, bool isDefaultAuthority)
+        public T WithAdfsAuthority(string authorityUri, bool validateAuthority)
```

Comment:
>validateAuthority [](start = 61, length = 17)

Can we have a default/override where this is true? both for aad and adfs?

Reply:
I've moved aad/adfs methods to be true for default of validateAuthority.